internal/exec/terraform.go

```diff
@@ -231,6 +231,14 @@ func ExecuteTerraform(info schema.ConfigAndStacksInfo) error {
 		}
 	}
 
+	// Check for any Terraform environment variables that might conflict with Atmos
+	for _, envVar := range os.Environ() {
+		if strings.HasPrefix(envVar, "TF_") {
+			varName := strings.Split(envVar, "=")[0]
+			u.LogWarning(atmosConfig, fmt.Sprintf("Found Terraform environment variable '%s' which may conflict with or be overwritten by Atmos", varName))
```

Comment:
```suggestion
			u.LogWarning(atmosConfig, fmt.Sprintf("detected '%s' set in the environment; this may interfere with Atmos's control of Terraform.", varName))
```